src/main/resources/db/migration/V2017_10_19_0915__Add_settlement_related_columns.sql

```diff
@@ -1,3 +1,3 @@
 ALTER TABLE claim
-  ADD COLUMN settlement JSONB,
+  ADD COLUMN partyStatement JSONB,
```

Comment:
can't update a migration file after it has been merged

Reply:
wow! That was accidentally :) I'll revert it.